person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age? || parent_permission
+  end
+
+  private
+
+  def of_age?
+    return true if is_of_age? >= 18
```

Comment:
## [OPTIONAL]

- [ ] Note that ruby always returns the last line of code in a method. I would suggest removing the `return` keyword as ruby will always do it for you.